### PR TITLE
diplay nama masjid from tetapan public api

### PR DESCRIPTION
**Pull request by naimhasim**
#115

### CODE REVIEW

**Comment by rohaizanr** [comment]
tak sesuai pakai API ni. esok saya akan explain kenapa...

https://github.com/Dev4w4n/e-masjid.my/issues/115#issuecomment-1986983267

**Comment by rohaizanr** [comment]
pending API @fairul82

**Comment by rohaizanr** [comment]
@naimhasim please re-sync main branch.

added public-web/src/service/Tetapan/TetapanMasjidApi.js

File: public-web/src/components/Home/Front.jsx [hunk 1]
```diff
@@ -1,15 +1,57 @@
+import { useEffect, useState } from "react";
+import { getTetapanNamaMasjid } from "../../service/Tetapan/TetapanMasjidApi";
+import { useEffect, useState } from "react";
+import { getTetapanNamaMasjid } from "../../service/Tetapan/TetapanMasjidApi";
```
Comment (rohaizanr) [inline public-web/src/components/Home/Front.jsx:4]: salah merge ni

**Review by rohaizanr — CHANGES_REQUESTED** [review]

File: public-web/src/components/Home/Front.jsx (hunk 1 above)
Comment (naimhasim) [reply public-web/src/components/Home/Front.jsx]: dah ok kot

**Comment by Dev4w4n** [comment]
@naimhasim kena re-sync branch ... ada changes tadi

**Review by rohaizanr — APPROVED** [review]
perfect!
terima kasih

**Pull request merged**